### PR TITLE
feat: unified input

### PR DESCRIPTION
**Pull request by 17yongga**

### CODE REVIEW

**Comment by 17yongga** [comment]
**What was done?**
Story: TRAD-98 " As a user, I would like to see a unified design for input boxes."

Created unified and better styling for input boxes and some of the buttons for UI pages.

**Testing**
Some of the main pages that need to be tested for this new UI change are:
1.        Login page
2.        Sign up page
3.        User chat input boxes
Pay attention to any styling display issues, or any malfunctions of input boxes.
Tested I have performed:
-          Inputting valid data and observe styling of text input
-          Input invalid data and see if error checking enabled for new input boxes
-          Click around the input text boxes to see if any unexpected styling change occurred

File: screens/SignUpScreen.tsx [hunk 1]
```diff
@@ -101,23 +110,43 @@ export default function SignUpScreen({ navigation }: any) {
...
+                <Button
+                    color="gray"
+                    mode="contained"
+                    contentStyle={styles.buttonContentStyle}
+                    style={styles.buttonContainer}
                     onPress={() => navigation.navigate('Login')}>
-                    <Text style={styles.postText}> Log in  </Text>
-                </TouchableOpacity>
+                    <Text style={styles.postText}> Log In  </Text>
+                </Button>
```
Comment (j584lee98) [inline screens/SignUpScreen.tsx:129]: I feel that maybe we can componentize the login and signup screens for these two buttons as they are being used again. That way, we only need to set a few additional attributes on these screens.

**Review by j584lee98 — COMMENTED** [review]
Great work Gary! These are some nice touches for our login and signup screens.

**Comment by ryanhow10** [comment]
Looks good, great work!

**Comment by alanwu4321** [comment]
Changes are merged in with https://github.com/TradeDis/frontend/pull/20

**Pull request closed without merging**